xknxproject/models/models.py

```diff
@@ -30,14 +31,20 @@ def __init__(
         self.project_uid = project_uid
         self.description = description
         self.dpt = dpt
+        self.comment = comment
 
         self.address = self._parse_address()
 
     def _parse_address(self) -> str:
         """Parse a given address and returns a string representation of it."""
-        main = (self.raw_address & 0b1111100000000000) >> 11
-        middle = (self.raw_address & 0b11100000000) >> 8
-        sub = self.raw_address & 0b11111111
+        return XMLGroupAddress.str_address(self.raw_address)
+
+    @staticmethod
+    def str_address(raw_address: int) -> str:
+        """Parse a given address and returns a string representation of it."""
+        main = (raw_address & 0b1111100000000000) >> 11
+        middle = (raw_address & 0b11100000000) >> 8
+        sub = raw_address & 0b11111111
```

Comment:
The only use of `_parse_address()` is 2 lines above in `__init__`. You can just use the staticmethod there too. No need to essentially copy the method.

Reply:
I wanted to keep the `_parse_address` method for compatibility reason. It is very lightweight and actually no real code duplication.

Reply:
It's marked private anyway.
And I think using staticmethods from `__init__` instead of a method is nicer anyway (no need for special attribute assignment order).